go.mod

```diff
@@ -4,6 +4,7 @@ go 1.13
 
 require (
 	cloud.google.com/go v0.56.0 // indirect
+	github.com/fatih/color v1.10.0 // indirect
```

Comment:
Is this going to get compiled in to every operator release?  If this is needed for tooling only - should we create a tools/ folder, with its own go.mod, etc.

Reply:
that's weird, wonder if i installed this by accident